src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/config/BackupProperties.java

```diff
@@ -0,0 +1,15 @@
+package com.rackspacecloud.metrics.ingestionservice.influxdb.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import java.time.Duration;
+
+@Component
+@Data
+@ConfigurationProperties("backup")
+public class BackupProperties {
+    private String gcsBackupBucket;
+    private Duration gcsTimeout;
```

Comment:
...not quite sure what constraints can be used on a `Duration`, but `@NotNull` would work at least.